site/content/api/Device-Registration-API.md

```diff
@@ -61,12 +61,14 @@ The body of the response message consists of a single *Data* section containing
 | Name             | Mandatory | JSON Type     | Description |
 | :--------------- | :-------: | :------------ | :---------- |
 | *device-id*      | *yes*     | *string*      | The ID of the device that is subject of the assertion. |
+| *gw-supported*   | *yes*     | *boolean*     | If set to `true`, the device allows one or more gateways to act on its behalf. These gateways are configured in the device registration data by means of the `via` property. |
```

Comment:
I wonder if we should make this optional with a default value of `false`. That way, a registry that doesn't support gateway mode at all would not need to implement it. WDYT?

Reply:
Well, as of now, the standard `BaseRegistrationService.getAssertionPayload()` method (marked final) will always add such a property anyway.
I can make the method used to get the value of the `gw-supported` property overridable, so that a device registry implementation could always return `false` there.
In that sense, I see not so much value in making the property optional, but have no problem with it either.

Reply:
Made `gw-supported` optional and made `isGatewaySupportedForDevice()` protected in `BaseRegistrationService`.